providers/implementations/ciphers/cipher_aes_cbc_hmac_sha.c

```diff
@@ -361,6 +368,12 @@ static void *aes_cbc_hmac_sha256_newctx(void *provctx, size_t kbits,
     return ctx;
 }
 
+static void *aes_cbc_hmac_sha256_dupctx(void *provctx)
+{
+    PROV_AES_HMAC_SHA256_CTX *ctx = provctx;
```

Comment:
nit: we usually put a blank line after declarations

Reply:
fixed

Reply:
It seems you didn't push

Reply:
Its done now, sorry about that